packages/engine-core/src/data-proxy/errors/BadRequestError.ts

```diff
@@ -0,0 +1,13 @@
+import { DataProxyAPIError } from './DataProxyAPIError'
+import type { RequestResponse } from '../utils/request'
+
+export class BadRequestError extends DataProxyAPIError {
+  public code = 'P5000'
```

Comment:
Errors related (only) to the data proxy client are now having error codes with `P5xxx`

Reply:
Are you going to document these classes and codes somewhere? We'd prefer to keep them in our own docs while we are in EAP (we may change them in the future, don't want to set a strong contract yet)